v3.9/getting-started/kubernetes/hardway/install-typha.md

```diff
@@ -4,7 +4,7 @@ redirect_from: latest/getting-started/kubernetes/hardway/install-typha
 canonical_url: 'https://docs.projectcalico.org/v3.9/getting-started/kubernetes/hardway/install-typha'
 ---
 
-**Typha** sits between the Kubernetes API server and per-node daemons like **Felix** and **confd** (running in `calico/node`).
+**Typha** is an optional add on to calico for performance optimization when you have a large number of nodes in a cluster.  It sits between the Kubernetes API server and per-node daemons like **Felix** and **confd** (running in `calico/node`).
```

Comment:
It's only optional until it's not.  At >50 nodes it really is required if you don't want your API server to melt.  This is because Felix needs some information from the Node resource and every node resource ges updated every second or two resulting in `O(n^2)` events to send (form the API server's point of view) as the cluster grows.